src/commands/install.ts

```diff
@@ -143,17 +144,41 @@ function getRollupReplaceKeys(env: EnvVarReplacements): Record<string, string> {
  * field instead of the CJS "main" field.
  */
 function resolveWebDependency(dep: string): DependencyLoc {
-  // if dep includes a file extension, check that dep isn't a package before returning
+  // if dep points directly to a file within a package, return that reference.
+  // No other lookup required.
   if (path.extname(dep) && !validatePackageName(dep).validForNewPackages) {
     const isJSFile = ['.js', '.mjs', '.cjs'].includes(path.extname(dep));
     return {
       type: isJSFile ? 'JS' : 'ASSET',
       loc: require.resolve(dep, {paths: [cwd]}),
     };
   }
+  // If dep is a path within a package (but without an extension), we first need 
+  // to check for an export map in the package.json. If one exists, resolve to it.
+  const packageName = getPackageNameFromSpecifier(dep);
+  if (packageName !== dep) {
+    const [packageManifestLoc, packageManifest] = resolveDependencyManifest(packageName, cwd);
+    if (packageManifestLoc && packageManifest && packageManifest.exports) {
+      const packageEntrypoint = `./${dep.replace(packageName + '/', '')}`;
+      const exportMapEntry = packageManifest.exports[packageEntrypoint];
+      const exportMapValue = exportMapEntry?.browser || exportMapEntry?.import || exportMapEntry?.default || exportMapEntry?.require || exportMapEntry;
+      if (typeof exportMapValue !== 'string') {
+        throw new Error(
+          `Package "${packageName}" exists but explicit package.json "exports" does not include valid "${packageEntrypoint}" entry.`,
```

Comment:
Nice!

Suggestion: add a section to the **Troubleshooting** section where people can search for this exact error message and there’s a longer explanation in the docs, with a link to how to set this up.